src/rpc/mempool.cpp

```diff
@@ -812,6 +813,39 @@ static RPCHelpMan savemempool()
     };
 }
 
+static RPCHelpMan getorphantxs()
+{
+    return RPCHelpMan{"getorphantxs",
+        "\nShows transactions from the tx orphanage.\n",
+        {},
+        {
+            RPCResult{"orphans",
+                RPCResult::Type::ARR, "", "",
+                {
+                    {RPCResult::Type::STR_HEX, "", "The transaction id"},
```

Comment:
For originator, I was thinking `fromPeer` (the NodeId). Currently each orphan tracks 1 sender, but hoping to have multiple in the future (#28031 if you're curious).

For this PR, I think it would suffice to take an approach similar to `getrawmempool` but with an integer verbosity.
verbose=0: list of txids (it's ok if there are duplicates, it should be rare anyway)
verbose=1: list of objects with txid, wtxid, size, senders (a list of NodeIds which as of now has length 1), expiration

I think a further verbose=2 could contain the transaction hex as well. There are more orphanage changes coming, which could add more fields. However I don't want to hold things up with bikeshedding, so feel free to decide what you think the scope of this PR should be.

Reply:
Thanks.  Updated with three verbosity levels (0, 1, and 2).